### ISSUE TITLE
Update the FDA submission date goal

### ISSUE THREAD

**Issue by dandv** [tidepool-org/tidepool-org.github.io#53]
At http://developer.tidepool.io/platform-roadmap/:

> Our very aggressive goal is to submit the appropriate application to the FDA by the end of 2014.

**Comment by jh-bate** [comment]
ping @brandonarbiter @HowardLook do we have an update to this timeframe now?

**Comment by HowardLook** [comment]
Good catch. We ended up meeting with the FDA in October 2014, notes here:
https://tidepool.org/wp-content/uploads/2014/09/001_Tidepool_FDA_Pre-Submission_2014-09_WebNoSig.pdf
https://tidepool.org/wp-content/uploads/2013/08/002_Tidepool_Meeting_Minutes.pdf

We will be Registering and Listing, but not filing a 510k based on their
guidance. Registering and Listing is a simple process, due soon (possibly
next sprint).

I will update the GitHub docs.
